Code/GraphMol/RWMol.cpp

```diff
@@ -298,7 +298,7 @@ unsigned int RWMol::addBond(unsigned int atomIdx1, unsigned int atomIdx2,
     dp_ringInfo->reset();
   }
 
-  return numBonds;//res;
+  return numBonds;  // res;
 }
 
 unsigned int RWMol::addBond(Atom *atom1, Atom *atom2, Bond::BondType bondType) {
```

Comment:
This makes me think we may need an "addBonds" as well to remove the N**2 iterations.